dist/accordion/css/accordion.min.css

```
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
+  visibility: visible;
+  -webkit-transition: max-height 800ms ease-in;
+  -moz-transition: max-height 800ms ease-in;
+  -ms-transition: max-height 800ms ease-in;
+  -o-transition: max-height 800ms ease-in;
+  transition: max-height 800ms ease-in;
```

```suggestion
  -webkit-transition:
    max-height 0.3s,
    padding-top 0.3s,
    padding-bottom 0.3s,
    opacity 0.5s;
  -o-transition:
    max-height 0.3s,
    padding-top 0.3s,
    padding-bottom 0.3s,
    opacity 0.5s;
  transition:
    max-height 0.3s,
    padding-top 0.3s,
    padding-bottom 0.3s,
    opacity 0.5s;
```
Last 3 browser versions is enough (IE9+10+11) regarding vendor prefixes. Also, we need `padding` and `opacity` transitions. No value specifies the default `ease` one for the speed curve ensuring a transition effect with a slow start, then fast, then end slowly.